backend/package.json

```diff
@@ -14,10 +14,12 @@
   "license": "ISC",
```

Comment:
Nothing for me to add that Aman didn't address for now. I will review for final approval once the requested changes are made.